services/ui-src/src/utils/autosave/autosave.ts

```diff
@@ -114,17 +112,14 @@ export const autosaveFieldData = async ({
   if (fieldsToSave.length) {
     const reportKeys = { reportType, id, state };
     let dataToWrite = {};
-    if (
-      entityContext &&
-      entityContext.selectedEntity &&
-      entityContext.entityType
-    ) {
+    if (entityContext && entityContext.selectedEntity) {
```

Comment:
Shorten to `entityContext?.selectedEntity` ?

Reply:
good call! ✅